### PR TITLE
Delete row

### PR DESCRIPTION
**Pull request by annasirunian**
https://app.zenhub.com/workspaces/bioviz-scrum-5d22fe4139b1324e0011234c/issues/dd-decaf/scrum/1052

New package: vuetify-confirm which allows to get confirmation from user when deleting condition or sample with related data.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
     "vue-moment": "^4.0.0",
     "vue-router": "^3.0.6",
     "vuetify": "^1.5.14",
+    "vuetify-confirm": "^2.0.0",
```
Comment (lukaw3d) [inline package.json]: @kvikshaug I think their abstraction of dialogs as a promise are worth it. We talked with Anna that we could copy a subset of their code and use it for input dialogs too

Comment (kvikshaug) [reply package.json]: Well, I'd be on board with the idea of forking the whole thing - we'll still get the abstraction and avoid the risks.

File: src/components/NewExperiment.vue [hunk 2]
```diff
@@ -238,7 +238,7 @@
                       </td>
                       <td>
                         <v-text-field
-                          v-model.number="fluxomicsItem.measurement"
```
[the remark this answers is not in the document]
Comment (annasirunian) [reply src/components/NewExperiment.vue]: There is a bug in `v-model.number`. When trying to enter negative decimal number between 0 and 1, it converts `-0` to `0`

File: src/components/NewExperiment.vue [hunk 3]
```diff
@@ -758,11 +877,112 @@ export default Vue.extend({
...
         temporaryId: uuidv4()
       });
     },
+    deleteCondition(index) {
+      const conditionId = this.tables.conditions.items[index].temporaryId;
+      if (
+        this.tables.samples.items.some(
+          sample =>
+            sample.condition && sample.condition.temporaryId === conditionId
+        )
```
[the remark this answers is not in the document]
Comment (annasirunian) [reply src/components/NewExperiment.vue]: Thanks for a great suggestion!

File: package.json (hunk 1 above)
Comment (annasirunian) [reply package.json]: Agreed with Luka to keep the package for now and replace it with our custom component later

**Pull request merged**